engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -103,6 +103,7 @@
 {
   public static final String DT_VERSION = StreamingApplication.DT_PREFIX + "version";
   public static final String DT_DFS_ROOT_DIR = StreamingApplication.DT_PREFIX + "dfsRootDirectory";
+  public static final String APEX_DFS_ROOT_DIR = StreamingApplication.APEX_PREFIX + "dfsRootDirectory";
```

Comment:
Okay, this changes the spec we had in the JIRA but I'll do it (if there are no objections)